charforge-gui/backend/app/services/settings_service.py

```
@@ -0,0 +1,131 @@
+from sqlalchemy.orm import Session
+from typing import Dict, Optional
+import base64
```

Ruff (0.12.2)

`base64` imported but unused

Remove unused import: `base64`

(F401)